api/metrics/rideshare_test.1.py

```diff
@@ -2,6 +2,7 @@
 sys.path.append("../")
```

Comment:
Looks like the file `api/metrics/rideshare_test.1.py` was added by accident in one of Tabor's previous PRs and I missed that in my review. Can you delete this file?

Are the tests you added in this file identical to the other test file you added? If so, you can just delete this file, otherwise you can move the new tests over.